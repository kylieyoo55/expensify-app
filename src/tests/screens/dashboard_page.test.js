import React from 'react';
import {shallow} from 'enzyme';
import DashBoard from '../../screens/Dashboard'
import toJSON from 'enzyme-to-json';

test('should render dashboard page correctly',()=>{
    const wrapper=shallow(<DashBoard />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})