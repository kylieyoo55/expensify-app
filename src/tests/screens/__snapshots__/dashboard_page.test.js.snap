// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page correctly 1`] = `
<div>
  <h1>
    Dash Board
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;

